sections/introduction.tex

```diff
@@ -0,0 +1,64 @@
+By modeling this relationship, connections between authors and topics, and between different authors, can be inferred giving more underlying information to the topics.
+They also show that the Author-Topic combination yields better and more coherent topics, which begs the question of whether any other document-related data can be applied similarly.
+However, they only test their algorithm on scientific papers, where usually the authors write about the same subject (their research field), which might not be the case for other fields, like journalism. \vejleder{Ikke helt holdbart! På den anden side så ser man nok mere spredning i ?? inden du nyheder + news + forfattere}
+
+% Intro to Nordjyske data and applications
+We have a dataset from a danish media group, called Nordjyske, with three years of article data.
+The dataset contains a variety of different metadata, which have the potential to improve topic models in the same way as \citet{author_topic_2012}. 
+In this paper, the dataset will be used with a focus on topic modeling and incorporation of metadata.
+
+% Intro to meta data
+The dataset used with a topic model can have a large impact on the topics generated.
+When referring to a dataset, we talk about a dataset including metadata information.
+One example of this could be the publication date of a text document. 
+In most contexts, it is not needed to use and understand the document; however, it provides more context to the main content of the document: the text.
+However, a lot of metadata is not used, either because it is not useful or we do not know how to use it properly. \vejleder{lidt ude af kontekst den her paragraf}
+
+% In this paper
+We want to construct different topic models that incorporate various metadata, to see whether incorporating this information changes the topic quality. \vejleder{hvorfor ikke blot integrate and extract models to include meta-info. Som vi gør i problemformuerlingen}
+We also investigate differences between the produced topics for various models, to evaluate their potential uses.
+
+% Problem Statement
+We define the following research questions to investigate in this paper:
+
+\begin{itemize}
+	\item \textit{How can we establish models that incorporate metadata from the Nordjyske dataset?}
+	\item \textit{How does including metadata within such models impact the resulting topics?}
+\end{itemize}
+
+% Contributions
+Our work is similar in goal to that of \citet{MetaLDA2017} since we work with meta information and are applying it to an \gls{lda} model.
+However, \citet{MetaLDA2017} learn a specific Dirichlet prior based on the meta-information given in their dataset, which gives a specific topic distribution for each meta-information field and in turn affect the document-topic distribution used with the \gls{lda}.
+Instead of using the document-topic distribution for drawing word topics, we want to create a new meta-topic distribution which influences the drawn topics.
```

Comment:
```suggestion
Instead of using the document-topic distribution for drawing word topics, we want to create a new metadata-topic distribution which influences the drawn topics.
```

Reply:
```suggestion
Instead of using the document-topic distribution for drawing word topics, we want to create a new metadata-topic distribution that influences the drawn topics.
```